tests/app/billing/test_billing.py

```diff
@@ -282,118 +284,90 @@ def test_create_free_sms_fragment_limit(client, sample_service):
+                           data=json.dumps(data_new),
+                           headers=[('Content-Type', 'application/json'), create_authorization_header()])
 
     response_get = client.get(
-        # Not specify a particular year to return all data for that service
-        'service/{}/billing/free-sms-fragment-limit'.format(sample_service.id),
+        'service/{}/billing/free-sms-fragment-limit?financial_year_start={}'
+        .format(sample_service.id, current_year),
         headers=[('Content-Type', 'application/json'), create_authorization_header()])
-    json_resp = json.loads(response_get.get_data(as_text=True))
-    assert response_get.status_code == 200
-    assert len(json_resp['data']) == 3
-    print(json_resp)
-    for i in [0, 1, 2]:
-        assert json_resp['data'][i]['free_sms_fragment_limit'] == limits[i]
-        assert json_resp['data'][i]['financial_year_start'] == years[i]
-
 
-def test_get_free_sms_fragment_limit_no_year_data_return_404(client, sample_service):
-
-    response_get = client.get(
-        'service/{}/billing/free-sms-fragment-limit?financial_year_start={}'.format(sample_service.id, 1999),
-        headers=[('Content-Type', 'application/json'), create_authorization_header()])
     json_resp = json.loads(response_get.get_data(as_text=True))
 
-    assert response_get.status_code == 404
+    assert response.status_code == 201
+    assert response_get.status_code == 200
+    assert json_resp['financial_year_start'] == current_year
+    assert json_resp['free_sms_fragment_limit'] == 9999
+
 
+def test_get_free_sms_fragment_limit_current_year(client, sample_service):
 
-def test_get_free_sms_fragment_limit_unknown_service_id_return_404(client):
+    current_year = get_current_financial_year_start_year()
+    create_annual_billing(sample_service.id, 9999, current_year - 1)
 
     response_get = client.get(
-        'service/{}/billing/free-sms-fragment-limit'.format(uuid.uuid4()),
+        'service/{}/billing/free-sms-fragment-limit'.format(sample_service.id),
         headers=[('Content-Type', 'application/json'), create_authorization_header()])
+
     json_resp = json.loads(response_get.get_data(as_text=True))
-    assert response_get.status_code == 404
+    assert response_get.status_code == 200
+    assert json_resp['financial_year_start'] == get_current_financial_year_start_year()
+    assert json_resp['free_sms_fragment_limit'] == 250000
 
 
-def test_get_free_sms_fragment_limit_current_year(client, sample_service):
-    response = client.get(
-        'service/{}/billing/free-sms-fragment-limit/current-year'.format(sample_service.id, True),
-        headers=[('Content-Type', 'application/json'), create_authorization_header()])
-    json_resp = json.loads(response.get_data(as_text=True))
+def test_get_free_sms_fragment_limit_past_year_not_exist(client, sample_service):
+    current_year = get_current_financial_year_start_year()
+    create_annual_billing(sample_service.id, 9999, current_year - 1)
```

Comment:
Better to put in the keyword arg in the parameter so it's easier to understand what `9999` means.

Reply:
To make it clearer this could be -

`create_annual_billing(sample_service.id, free_sms_fragment_limit=9999, financial_year_start=current_year - 1)`

as it is much clearer